owasp-dependency-check-suppressions.xml

```diff
@@ -8,4 +8,46 @@
       <packageUrl regex="true">^pkg:maven/org\.apache\.maven/maven\-core@.*$</packageUrl>
       <cve>CVE-2021-26291</cve>
    </suppress>
-</suppressions>
\ No newline at end of file
+   <suppress>
+      <notes><![CDATA[
```

Comment:
This should have been in a separate pull request since it's for a different vulnerability.